src/pages/workspace/members/WorkspaceMemberDetailsPage.tsx

```diff
@@ -0,0 +1,147 @@
+    const styles = useThemeStyles();
+    const theme = useTheme();
+    const {translate} = useLocalize();
+
+    const [removeMemberConfirmModalVisible, setRemoveMemberConfirmModalVisible] = React.useState(false);
+
+    const accountID = Number(route?.params?.accountID) ?? 0;
+    const backTo = route?.params?.backTo;
+
+    const member = policyMembers?.[accountID];
+    const details = personalDetails?.[accountID] ?? ({} as PersonalDetails);
+    const avatar = details.avatar ?? UserUtils.getDefaultAvatar();
+    const fallbackIcon = details.fallbackIcon ?? '';
+    const displayName = details.displayName ?? '';
+
+    const askForConfirmationToRemove = () => {
+        setRemoveMemberConfirmModalVisible(true);
+    };
+
+    const removeUser = useCallback(() => {
+        Policy.removeMembers([accountID], route.params.policyID);
+        setRemoveMemberConfirmModalVisible(false);
+        Navigation.goBack(backTo);
+    }, [accountID, backTo, route.params.policyID]);
+
+    const redirectToProfile = useCallback(() => {
+        Navigation.navigate(ROUTES.PROFILE.getRoute(accountID));
+    }, [accountID]);
+
+    const openRoleSelectionModal = useCallback(() => {
+        Navigation.navigate(ROUTES.WORKSPACE_MEMBER_ROLE_SELECTION.getRoute(route.params.policyID, accountID));
+    }, [accountID, route.params.policyID]);
+
+    return (
+        <ScreenWrapper testID={WorkspaceMemberDetailsPage.displayName}>
+            <FullPageNotFoundView>
+                <HeaderWithBackButton
+                    title={displayName}
+                    subtitle={policy?.name}
+                    onBackButtonPress={() => Navigation.goBack(backTo)}
+                />
+                <View style={[styles.containerWithSpaceBetween, styles.pointerEventsBoxNone, styles.justifyContentStart]}>
+                    <View style={styles.avatarSectionWrapper}>
+                        <OfflineWithFeedback pendingAction={details.pendingFields?.avatar}>
+                            <Avatar
+                                containerStyles={[styles.avatarXLarge, styles.mv5, styles.noOutline]}
+                                imageStyles={[styles.avatarXLarge]}
+                                source={UserUtils.getAvatar(avatar, accountID)}
+                                size={CONST.AVATAR_SIZE.XLARGE}
+                                fallbackIcon={fallbackIcon}
+                            />
+                        </OfflineWithFeedback>
+                        {Boolean(details.displayName ?? '') && (
+                            <Text
+                                style={[styles.textHeadline, styles.pre, styles.mb6, styles.w100, styles.textAlignCenter]}
+                                numberOfLines={1}
+                            >
+                                {displayName}
+                            </Text>
+                        )}
+                        <Button
```

Comment:
NAB: Kindly confirm if the spacing between the remove button and the role is correct? Since it seems having more space.

<img width="464" alt="Screenshot 2024-03-06 at 5 33 55 PM" src="https://github.com/Expensify/App/assets/46707890/d9888227-b114-4014-ae54-6d0b57f88302">

Reply:
right - there is too much space

Reply:
fixed